sdk/tests/deqp/modules/shared/glsFboUtil.js

```diff
@@ -1003,7 +1003,7 @@ goog.scope(function() {
     * @return {number}
     */
     glsFboUtil.formatkey = function(format, type) {
-        return (type << 16 | format) & 0xFFFFFFFF;
+        return ((type << 16) >>> 0 | format) & 0xFFFFFFFF;
```

Comment:
Can you add a comment here what the intension of ">>> 0" is here? Is it to convert from signed 64 bit to unsigned 32 bit?